runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -357,7 +359,7 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 
-		if (config.screenshotUntradeableDrop())
+		if (config.screenshotUntradeableDrop() && !isInsideGauntlet())
```

Comment:
Would there be any reason to want your gauntlet untradeables screenshotted, ie should this be configurable?

Reply:
No as EVERY single item dropped by them is untradeable and generates a screenshot for EACH item/item stack. I killed ONE npc and got 3 screenshots. You kill multiple every single run, therefore you start loading up on screenshots that are absolutely useless.